facades.md

```diff
@@ -0,0 +1,49 @@
+# Facade data in third-party-web
+
+In addition to identifying entities, third-party-web also includes data on entities' _products_, and available alternative libraries for products known as _facades_. A **third-party product** is a third-party resource that is included on a page, for example: a YouTube embed iframe. A **facade** is a frontend element which looks similar to the actual product and lazily loads it upon user interaction.  It's expected to be significantly more lightweight/faster. For example: [lite-youtube-embed](https://github.com/paulirish/lite-youtube-embed).
+
+[Lighthouse](https://github.com/GoogleChrome/lighthouse) 7.0 has a new audit that identifies third-party 
+embeds (link coming soon) that can be lazy loaded with a facade. The Lighthouse audit is powered by the facade data in third-party-web.
+
+## Criteria for adding a new facade
+### Basic functionality
+
+* Loads an HTML component on page load which looks like the actual third-party 
```

Comment:
Imo, a facade should present an approximate of the actual third-party, but it doesn't strictly need to load an HTML component to do this. It's valid if you have an image/screenshot/generated scrape of the 3P that loads the real thing on interaction.

Something like https://twitter.com/wongmjane/status/1330273157245243394 is an example of where an image can be fine here.

Reply:
Isn't your example an HTML component?

I interpreted "HTML component" here to mean anything that shows up in the DOM as compared to a library/utility that only exists in JS.

Reply:
yeah i was going with a loose definition of "component". There doesn't feel like there's a great word for this thing. It's not a "library".... even "widget" is close but doesn't feel right.

Reply:
Patrick: after taking a second look, you're of course correct. I know finding the right wording here is tricky and so HTML component may be totally fine if its to be interpreted with a loose definition.

Reply:
If it took this conversation to explain thats what we're talking about to someone already very familiar with the area though, perhaps thats enough of a signal that others might not understand what we're talking about :)

Is there an alternative term you had in mind @addyosmani that has a better blend of the generic yet specific we need here?